auth-connector/src/main/proto/usermanagement.proto

```diff
@@ -1703,6 +1727,13 @@ message ListMachineUsersRequest {
   bool includeInternal = 5;
   // Whether to include deleted machine users.
   bool includeDeleted = 6;
+  // Whether to include workload machine users. All regular machine users are
+  // workload machine users. This is a subset of the internal machine users,
+  // which have a password set. This field has no effect if internal machine
+  // users are requested because all workload machine users will already be
+  // included in the results. Note that this has no impact when listing by name
+  // or CRN. For those calls, internal machine users are always returned.
+  bool includeWorkloadMachineUsers = 7;
```

Comment:
this is the needed proto change. the rest of the proto is just to keep it in sync with the UMS.